### PR TITLE
feat(new pane): add new pane design and behaviors

### PR DESCRIPTION
**Pull request by JeanBarriere**
- [x] remove conditional panes in store
- [x] ensure library is first pane to show
- [x] improve panes toggling
- [x] remove open/closing modes
- [x] redesign diagram/toc panes
- [x] redesign pane toggler component
- [x] unit tests

---

### Acceptance

**When** I navigate to a Story
**Then** I see the new panel design

This new panel design is implemented in light of horizontal space being limited causing overlaps with cards (errors, suggestions, etc). Also, no panels work together in a way requiring them to be open together at any given moment.

**When** I toggle Viewing mode
**Then** the Table of Contents toggles in.

**When** I toggle Editing or Suggesting mode
**Then** the Library toggles in.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -135,11 +135,13 @@ module.exports = {
...
       spacing: {
         initial: 'initial',
+        inherit: 'inherit',
         'fit-content': 'fit-content',
         'max-content': 'max-content',
         '2full': '200%',
         14: '3.5rem', // 56px
         22: '5.5rem', // 88px
+        75: '18.75rem', // 300px
         76: '19rem' // 304px
```
Comment (Arinono) [inline tailwind.config.js]: Is the difference between 300px and 304px really making a big change ? Maybe we can use only one, like a multiple of 4, as we always did 🌝

File: tailwind.config.js [hunk 2]
```diff
@@ -157,9 +159,15 @@ module.exports = {
...
+      minWidth: {
+        inherit: 'inherit'
+      },
+      minHeight: {
+        inherit: 'inherit'
+      },
       maxWidth: {
         xxs: '10rem',
-        pane: '19rem'
+        pane: '18.75rem'
```
Comment (Arinono) [inline tailwind.config.js]: If you want to stick with exactly 300px, that's fine, but we should remove the `w-76` then.

File: tests/unit/app/views/DiagramPanel/index.spec.ts [hunk 3]
```diff
@@ -1,78 +1,22 @@
...
-        vm.toggled = false
-        vm.togglePanel()
-        expect(vm.toggled).toBeTruthy()
-        await new Promise(resolve => setTimeout(resolve, 400))
-        expect(vm.toggled).toBeFalsy()
-      })
+  describe('should mount', () => {
+    it('should have html', () => {
+      expect(view.html()).toBeTruthy()
     })
```
Comment (Arinono) [inline tests/unit/app/views/DiagramPanel/index.spec.ts]: The `view.destroy()` is missing (missed the last time)

File: tailwind.config.js [hunk 4]
```diff
@@ -157,9 +159,15 @@ module.exports = {
...
       }
```
Comment (Arinono) [inline tailwind.config.js]: We should keep only one of the two `w-75` `w-76`, they are really close and we don't need both. Maybe we should keep the `w-76` as it's a multiple of 4, and we already agreed on that previously (and I don't think the difference is that big).

File: tests/unit/app/views/DiagramPanel/index.spec.ts [hunk 5]
```diff
@@ -1,78 +1,22 @@
...
-      it('should set toggled to true for 350ms', async () => {
-        vm.toggled = false
-        vm.togglePanel()
-        expect(vm.toggled).toBeTruthy()
-        await new Promise(resolve => setTimeout(resolve, 400))
-        expect(vm.toggled).toBeFalsy()
-      })
+  describe('should mount', () => {
+    it('should have html', () => {
+      expect(view.html()).toBeTruthy()
```
Comment (Arinono) [inline tests/unit/app/views/DiagramPanel/index.spec.ts]: WEE WOO 🚨 
There no `view.destroy()` we missed it the last time.

File: tailwind.config.js (hunk 1 above)
Comment (JeanBarriere) [reply tailwind.config.js]: Yeah, I just followed design here. I can put back to 76

File: tests/unit/app/views/DiagramPanel/index.spec.ts (hunk 5 above)
Comment (JeanBarriere) [reply tests/unit/app/views/DiagramPanel/index.spec.ts]: :astonished:

**Comment by JeanBarriere** [comment]
@Arinono review addressed

**Pull request merged**